cljfmt/test/cljfmt/config_test.clj

```diff
@@ -1,10 +1,32 @@
 (ns cljfmt.config-test
   (:require [cljfmt.config :as config]
-            [clojure.test :refer [deftest is]]))
+            [clojure.test :refer [deftest is testing]]
+            [clojure.java.io :as io])
+  (:import (java.io File)))
```

Comment:
I believe in other files `(import [java.io File])` is used as the style.

Reply:
leftover import, will just yank